cli/enrol.yaml

```diff
@@ -0,0 +1,72 @@
+## Every recipe file must provide at least the component property.
+## Based on the plugin type plugin specific files files will be generated.
+component: enrol_plugintest
+
+## Human readable name of the plugin.
+name: Enrol plugin
+
+## Human readable release number.
+release: "0.1.0"
+
+## Plugin version number, e.g. 2016062100. Will be set to current date if left empty.
+#version: 2016121200
+
+## Required Moodle version, e.g. 2015051100 or "2.9".
+requires: "2.9"
+
+## Plugin maturity level. Possible options are MATURIY_ALPHA, MATURITY_BETA,
+## MATURITY_RC or MATURIY_STABLE.
+maturity: MATURITY_BETA
+
+## Copyright holder(s) of the generated files and classes.
+copyright: 2016 Alexandru Elisei <alexandru.elisei@gmail.com>, David Mudrák <david@moodle.com>
+
+## Features flags can control generation of optional files/code fragments.
+features:
+    readme: true
+    license: true
+    install: true
+    settings: true
+
+## Allows manual enrolment. Requires enrol/<enrolname>:enrol capability.
+allow_enrol: 'false'
```

Comment:
We won't run into a conflict if we use `<plugintype>`. The component types are unique.

If a plugin type `features` is introduced then the current `features` section of the plugin will have to be renamed. Are you proposing that we should rename it now in case of a conflict?